libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -179,6 +185,39 @@ class PostgresVectorStore extends VectorStore {
     )
   }
 
+  static async fromTexts(_texts: string[], _metadatas: object[], _embeddings: Embeddings, _dbConfig: dbConfigArgs): Promise<VectorStore> {
```

Comment:
I prefer to use variables without the "_" prefix. 
I believe we should be using the more generic interface of EmbeddingsInnterface instead of the class Embeddings to ensure more flexibility.